From 6aebaa6e63a0a5d391c8f31b8ef3122af78b3d14 Mon Sep 17 00:00:00 2001
Message-Id: <6aebaa6e63a0a5d391c8f31b8ef3122af78b3d14.1571993010.git.keller@fs-net.de>
In-Reply-To: <e61320c6d86a362e7637cac37bf3ccdfa511838e.1571993010.git.keller@fs-net.de>
References: <e61320c6d86a362e7637cac37bf3ccdfa511838e.1571993010.git.keller@fs-net.de>
From: bings <bings@codeaurora.org>
Date: Wed, 28 Nov 2018 17:41:35 +0800
Subject: [PATCH 2/2] qcacld-2.0: Don't define pci related api if HIF_PCI isn't
 defined

If HIF_PCI is not defined, do not define pci related api, otherwise
there will be build error

Change-Id: I3d6989ea2ee4c7c13ad011b6b1ca11a1123de168
CRs-Fixed: 2357878
---
 CORE/VOSS/inc/vos_cnss.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CORE/VOSS/inc/vos_cnss.h b/CORE/VOSS/inc/vos_cnss.h
index 8bfae8619..51e1eaf31 100644
--- a/CORE/VOSS/inc/vos_cnss.h
+++ b/CORE/VOSS/inc/vos_cnss.h
@@ -51,12 +51,13 @@ enum cnss_cc_src {
 	CNSS_SOURCE_11D,
 	CNSS_SOURCE_USER
 };
-
+#ifdef HIF_PCI
 static inline void vos_wlan_pci_link_down(void){ return; }
 static inline int vos_pcie_shadow_control(struct pci_dev *dev, bool enable)
 {
 	return -ENODEV;
 }
+#endif
 
 static inline u8 *vos_get_cnss_wlan_mac_buff(struct device *dev, uint32_t *num)
 {
-- 
2.14.5

